app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,8 @@
+class DishIngredientsController < ApplicationController
+  
+  def create
+    di = DishIngredient.create!(dish_id: params[:dish_id], ingredient_id: params[:ingredient_id])
```

Comment:
could also create the dish_ingredient_params method to cut down on all the parameters here.